check_pgactivity

```diff
@@ -1917,6 +1919,9 @@ sub check_backends_status {
     delete $status{'idle in transaction (aborted)'}
         if $hosts[0] < $PG_VERSION_90;
 
+    delete $status{'other wait event'}
+        if $hosts[0] < $PG_VERSION_96;
```

Comment:
Congrats ! You probably found a bug in the existing check_pga code :)

It should probably be `$hosts[0]->{'version_num'}`, not just `$hosts[0]`